### ISSUE TITLE
Hitting 'Enter' in certain in-game interactions with Key Remapping enabled also open the chat.

### ISSUE THREAD

**Issue by treciphico** [runelite/runelite#13216]
I use key remapping primarily to change certain F-keys to WASD. By enabling Key remapping, the chatbox becomes 'Press Enter to Chat...'. During certain game interactions which require the use of the 'Enter' key, such as editing Grand Exchange offers, withdrawing X quantities of items from the bank, or responding to private messages, the 'Press Enter to Chat...' chatbox will to trigger, forcing me to press enter again to clear it and use my remapped keys.

1. Go to Settings, Key Remapping and enable it.
2. Go to Grand Exchange, find an item and click on it.
3. Modify the purchase price, hit Enter.
4. See that the 'Press Enter to Chat' in the chatbox is now gone and when you type it will be in the chat.

Expected behavior:
For the 'Enter' key to not trigger the chatbox in situations listed below with key remapping enabled.

Times I have encountered this issue: 
 - Editing grand exhange offers (quantity, price).
 - Withdraw-X/Deposit-X items from the bank.
 - Buy-X/Sell-X items from shops.
 - Responding to private messages.
 - Answering clue challenges.
 - Editing Runelite bank tags.
 - Setting up autochat.
 - Adding/deleting friends.
 - Joining clan chats.
 - Adding charges to blowpipe, trident/toxic trident, serpentine/tanzanite/magma helm, sanguinesti staff, scythe of vitur.
 - Adding/removing pages to tome of fire, teleports to master scroll book, ectotokens to bonecrusher/necklace, ancient shards to arclight.
 - Entering how many house tabs to redirect with redirection scroll.
 - Entering quantity of gp to deposit/withdraw from coffers/duels.
 - Remove-X/Store-X at tool leprechuans and seed vault.
 - Storing X items in looting bag.
 - Crafting X amount of jewellery at a furnace.
 - Charging crystal equipment(bow, shield, armour, blade of saeldor, axe, pickaxe, harpoon) with crystal shards.
 - Making X amount of Dorgesh-kaan or Goblin Village spheres.
 - Withdrawing quantites of overload, absorption from nightmare zone, purchasing as well.
 - Withdrawing X items from Chambers of Xeric chests.
 - Buying X items from Theatre of Blood chests.
 - Withdrawing X items from gravestone or NPC storing items lost on death.

**Basically anything with a dialogue box that requires the use of the 'Enter' key will also trigger the 'Press Enter to Chat' chatbox,**

After testing, this issue does not occur during certain skilling actions such as fletching, smithing, and cooking with the X option in the redesigned skilling menu in the chatbox.

Environment:
 - OS: Windows 10 x64
 - RuneLite version: 1.7.0
 - Launcher version: 2.1.3

**Comment by Adam-** [comment]
Did this only recently start happening? Or has it always happened? And if so when?

**Comment by Mrnice98** [comment]
I just had a similar thing happen. I was just killing varrock guards and then tried to send a message i was able to type everything aside from the char's WASD. Was quite random as I was not using any interface just sending a regular public chat message.

**Comment by treciphico** [comment]
@Adam- This has always happened to me with Key remapping enabled. I used to used Corair iCUE software to remap keys, but decided to use the one build into Runelite and disabled all remapping features in iCUE. I just thought it was a minor inconvenience but decided to log all the instances in which it occured and post it here. With me, this happens with the F-key remapping, but might also occur with the WASD camera as in @Mrnice98's case.

**Comment by Adam-** [comment]
fixed in 8a728c1053e174bbbf15648446824e20fea99fe5

**Comment by treciphico** [comment]
Awesome, thank you! Appreciate it!
